src/structs/loaded/classfile_helper.rs

```diff
@@ -374,21 +349,20 @@ pub fn create_attributes(
         let name = const_pool.utf8(entry.attribute_name_index as usize)?;
         let name = Arc::clone(&name);
 
-        let data = match name.str.as_str() {
```

Comment:
This should be left as a match because there will be more items here in the future. Would be good to document this:

```rs
// TODO: Implement standard attributes.
```